src/main/java/me/namuhuchutong/ladder/domain/Row.java

```diff
@@ -1,60 +1,90 @@
+import static me.namuhuchutong.ladder.domain.wrapper.LadderExpression.*;
+import static me.namuhuchutong.ladder.domain.wrapper.Movements.*;
 
 public class Row {
 
-    private final List<LadderExpressionEnum> values;
+    private static final String BLANK = "";
+
+    private final List<LadderExpression> values;
 
     public static Row from(int participants, ScaffoldFactory factory) {
-        List<LadderExpressionEnum> collect = initializeLadderRow();
-        collect.addAll(addScaffold(participants, factory));
-        return new Row(unmodifiableList(collect));
+        List<LadderExpression> initializedRow = initializeLadderRow();
+        initializedRow.addAll(addScaffold(participants, factory));
+        return new Row(unmodifiableList(initializedRow));
     }
 
-    private static List<LadderExpressionEnum> addScaffold(int participants, ScaffoldFactory factory) {
-        List<LadderExpressionEnum> result = new ArrayList<>();
+    private static List<LadderExpression> addScaffold(int participants, ScaffoldFactory factory) {
+        List<LadderExpression> result = new ArrayList<>();
         for (int i = 1; i < participants; i++) {
             result.add(factory.createScaffold());
             result.add(VERTICAL_BAR);
         }
         return result;
     }
 
-    private static List<LadderExpressionEnum> initializeLadderRow() {
-        List<LadderExpressionEnum> list = new ArrayList<>();
+    private static List<LadderExpression> initializeLadderRow() {
+        List<LadderExpression> list = new ArrayList<>();
         list.add(EMPTY_SPACE);
         list.add(VERTICAL_BAR);
         return list;
     }
 
-    public Row(List<LadderExpressionEnum> values) {
+    public Row(List<LadderExpression> values) {
         validateContinuousScaffold(values);
         this.values = values;
     }
 
-    private void validateContinuousScaffold(List<LadderExpressionEnum> values) {
+    private void validateContinuousScaffold(List<LadderExpression> values) {
         ContinuousScaffoldValidator validator = new ContinuousScaffoldValidator();
         values.stream()
-              .filter(expression -> !expression.equals(VERTICAL_BAR))
+              .filter(this::filteringVerticalBar)
               .filter(validator::isContinuous)
               .findAny()
               .ifPresent(e -> {
                   throw new IllegalArgumentException("사다리 발판은 연속적일 수 없습니다.");
               });
     }
 
+    private boolean filteringVerticalBar(LadderExpression expression) {
+        return expression.equals(HYPHEN);
+    }
+
+    public int move(int startPoint) {
+        Movements movement = findDestination(startPoint);
+        return movement.move(startPoint);
+    }
+
+    private Movements findDestination(int startPoint) {
+        LadderExpression left = this.values.get(startPoint - 1);
+        LadderExpression right = isEndOfCollection(startPoint) ?
+                this.values.get(startPoint) : this.values.get(startPoint + 1);
```

Comment:
```suggestion
        LadderExpression right = isEndOfCollection(startPoint) ?
                this.find(startPoint) : this.find(startPoint + 1);
```

과할수도 있지만, 저는 `values`를 꺼내오는 함수를 만들어서 데이터 접근하는 곳을 한곳으로 모을것 같아요. 🤔 
관점 공유정도로 봐주세요~